traits/mlcontrol/partials/_locale_copy.htm

```diff
@@ -0,0 +1,21 @@
+<!-- Language Copy Dropdown -->
+<div class="dropdown ml-copy-dropdown">
+    <button
+        class="icon-copy btn ml-copy-btn"
+        data-toggle="dropdown"
+        type="button">
+    </button>
+    <ul class="dropdown-menu pull-right ml-copy-dropdown-menu" role="menu">
+        <?php foreach ($locales as $code => $name): ?>
+        <li role="presentation">
+            <a
+                role="menuitem"
+                tabindex="-1"
+                data-copy-locale="<?= $code ?>"
+                href="javascript:;">
+                Copy from <?= $name ?>
```

Comment:
```suggestion
                <?= e(trans('winter.translate::lang.locale.copy_from', ['locale' => $name])); ?>
```
then add the `copy_from` key to `lang/en/lang.php` under the `locale` key with the value `Copy from :locale`